test/secret_key_finder_test.rb

```diff
@@ -32,6 +32,25 @@ def config
   end
 end
 
+
+class Rails52SecretKeyBase
+  def credentials
+    OpenStruct.new(secret_key_base: nil)
+  end
+
+  def secrets
+    OpenStruct.new(secret_key_base: nil)
+  end
+
+  def config
+    OpenStruct.new(secret_key_base: nil)
+  end
+  
+  def secret_key_base
+    'secret_key_base' if Rails.env.development? || Rails.env.test?
```

Comment:
Why do we need this `if` statement here?

Reply:
Oh! How did I missed that! I was testing something else and I just left them there. I'll fix and squash now.